crates/wdk-build/src/utils.rs

```diff
@@ -26,6 +26,30 @@ use windows::{
 
 use crate::{ConfigError, CpuArchitecture};
 
+/// Type for versions of the format "major.minor"
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
+pub struct BasicVersion {
+    /// The major version number
+    pub major: u32,
+    /// The minor version number
+    pub minor: u32,
```

Comment:
Consider using a tuple struct instead of named fields .e.g. `BasicVersion(u32, u32)`